### PR TITLE
Fixed links to Smartrest guide

### PR DESCRIPTION
**Pull request by JoshuaAdemola**

### CODE REVIEW

**Review by BeateRixen — APPROVED** [review]

File: content/reference/device-management-library-bundle/overview.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ In the following section you will find descriptions of the most important functi
 
 If you are interested in details on exposing the {{< product-c8y-iot >}}´s functionalities through our Rest API, see the [{{< openapi >}}](https://{{< domain-c8y >}}/api/) for further information.
 
-Moreover, see [SmartREST](/reference/smartrest) for more information on SmartREST and a complete list of all SmartREST templates mentioned throughout the following sections.
+Moreover, see [SmartREST](/reference/smartrest-one) for more information on SmartREST and a complete list of all SmartREST templates mentioned throughout the following sections.
```
Comment (BeateRixen) [inline content/reference/device-management-library-bundle/overview.md:13]: I suppose that this should point to the SmartREST 2 section. @rssag what is your take?

Comment (rssag) [reply content/reference/device-management-library-bundle/overview.md:13]: yes, it's more up to date and should be used here

Comment (rssag) [reply content/reference/device-management-library-bundle/overview.md:13]: #data-format doesn't exist in smartrest-two. the rework is not yet done, the dev team needs to work on it at some point. i'd say we leave the link to smartrest-one here

Comment (BeateRixen) [reply content/reference/device-management-library-bundle/overview.md:13]: @JoshuaAdemola Please change the link here and in the other 2 PRs

**Pull request merged**